.gitignore

```diff
@@ -1,29 +1,3 @@
-# ignore everything
-*
 node_modules/
-# but descend into directories
-!*/*
-# recursively allow files under subtree
-!/test/**
-!/static/**
-!/assessments/mkdocs.yml
-!/bin
-!/ci_bin
-!/commands/**
-!/lib/**
-!/util/
-!.dockerignore
-!.editorconfig
-!.eslint*
-!.gitignore
-!.nvmrc
-!.travis.yml
-!docker-mkdocs.yml
-!Dockerfile
-!LICENSE
-!Makefile
-!package.json
-!README.md
-!relocatable_venv
-!yarn.lock
-
+.nyc_output/
+package-lock.json
```

Comment:
this refactor will include output files from the policy, such as `partials`, `docs`, etc. that we don't want